requirements.txt

```diff
@@ -14,3 +14,4 @@ python-openstackclient
 gitpython
 paramiko
 setuptools
+podman-compose
```

Comment:
Can we also add docker-compose

Reply:
Thanks to pointers from @paigerube14 around not needing to use the host network for the containers. Both podman-compose, docker-compose should work now.